### PR TITLE
🦋 Implement empty state in outbox

### PR DESCRIPTION
**Pull request by josemigallas**
[THREESCALE-9860: Implement EmptyState (Outbox)](https://issues.redhat.com/browse/THREESCALE-9860)

Before:
Screenshot 2023-10-10 at 11 03 23

After:
Screenshot 2023-10-11 at 09 42 13

### CODE REVIEW

File: app/views/provider/admin/messages/outbox/index.html.slim [hunk 1]
```diff
@@ -1,39 +1,49 @@
 - content_for :page_header_title, 'Sent Messages'
 
 - if @messages.blank?
-  p= t('provider.admin.messages.no_messages')
+  div class="pf-c-empty-state"
+    div class="pf-c-empty-state__content"
+      i class="fas fa-rocket pf-c-empty-state__icon" aria-hidden="true"
```
Comment (mayorova) [inline app/views/provider/admin/messages/outbox/index.html.slim:6]: Suggesting https://fontawesome.com/v4/icon/paper-plane-o

(no "outbox" icon it seems, GMail uses a paper plan for "Sent", so inspired by that)

Comment (josemigallas) [reply app/views/provider/admin/messages/outbox/index.html.slim:6]: Perfect 👍🏻

**Comment by josemigallas** [comment]
> The "empty" page looks good. What's strange is that for some reason I don't see my "sent" messages when I do send them 🙃
> 
> But that is either my local env problem, or it's broken somewhere else...

Yes I've seen the same, don't know why exactly it happens. Creating messages manually in the console do make the table render so it's got to be something else.

**Comment by mayorova** [comment]
> Yes I've seen the same, don't know why exactly it happens. Creating messages manually in the console do make the table render so it's got to be something else.

Yeah, I think I figured out! I think my Sidekiq was pointing to a different DB :upside_down_face: 

It is weird! So, Rails creates a Sidekiq job with the message details, but it seems that it's the job execution that persists the message in the DB! This is weird IMO... but well...

**Pull request merged**